src/components/MapView/MapView.tsx

```diff
@@ -52,18 +52,88 @@ const FIT_BOUNDS_OPTIONS = {
   padding: 20
 }
 
+type TableRowProps = {
```

Comment:
I think this should probably go in it's own component called tooltip or something

Reply:
Yeah I think the tooltip functions should move to their own folder, wasnt exactly sure how to structure though